readme.md

```diff
@@ -1,7 +1,7 @@
 # Product cards
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://Pavlo-lab-max-Kopalko.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://Pavlo-lab-max-Kopalko.github.io/layout_product-cards/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="950" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/9ecbee6a-ee13-4d78-97e0-edb2dfd6646f">

Reply:
I submited references ebow and here:
https://Pavlo-lab-max-Kopalko.github.io/layout_product-cards/
https://Pavlo-lab-max-Kopalko.github.io/layout_product-cards/report/html_report/